src/map/clif.c

```diff
@@ -4906,8 +4907,10 @@ static int clif_damage(struct block_list *src, struct block_list *dst, int sdela
 		unit->set_dir(src, unit->getdir(src));
 	}
 
+	// In case this assignment is bypassed by BDT_MULTICRIT
+	type = clif_calc_delay(type, div, damage + damage2, ddelay);
```

Comment:
this is designed to call ``clif_calc_delay`` twice if type is not BDT_MULTICRIT? or this is error?

Reply:
it's design to call twice